Initialize UI

SrinivasMushnoori opened the pull request.

kbsezginel committed:
Initialize command line interface.

kbsezginel committed:
Merge branch 'master' of https://github.com/SrinivasMushnoori/SSS_2017_QuESt

Commit:
Merge pull request #1 from kbsezginel/master

Initialize command line interface

dgasmith commented on quest.py:
```diff
@@ -0,0 +1,30 @@
+import argparse
```
Don't forget to add a line at the top of this file explaining what this is!

kbsezginel committed:
Merge branch 'master' of https://github.com/MolSSI-SSS/SSS_2017_QuESt

Commit:
Merge branch 'master' of https://github.com/MolSSI-SSS/SSS_2017_QuESt

dgasmith commented:
@SrinivasMushnoori Ping me when this is done. Doesn't have to be perfect, but lets get it in.

kbsezginel committed:
Merge branch 'master' of https://github.com/MolSSI-SSS/SSS_2017_QuESt

kbsezginel committed:
Add option to read molecule and parameters.

The molecule file is a text file that is converted to string and fed to Molecule class.
The input parameters is a yaml file which is read as a dictionary of dictionaries.
pyyaml is added to setup.py as a dependency.
A water molecule file is added to tests as a sample.
A sample parameters file is added o main directory.

kbsezginel committed:
Merge branch 'master' of https://github.com/MolSSI-SSS/SSS_2017_QuESt

Commit:
Merge branch 'master' of https://github.com/MolSSI-SSS/SSS_2017_QuESt

Commit:
Merge branch 'master' into master

Commit:
Merge pull request #2 from kbsezginel/master

Add molecule and parameters read to CLI.

Commit:
Merge branch 'master' of https://github.com/MolSSI-SSS/SSS_2017_QuESt

dgasmith commented on parameters.yml:
```diff
@@ -0,0 +1,21 @@
+qm:
```
This should likely be moved into the `quest` folder. Maybe title it `default_parameters.yml`.

Commit:
Merge branch 'master' of https://github.com/SrinivasMushnoori/SSS_2017_QuESt

dgasmith commented on quest.py:
```diff
@@ -0,0 +1,51 @@
+"""
```
This needs to be moved into the `quest` folder as well. We usually want to call it something `CLI` for command line interface.

SrinivasMushnoori commented:
@dgasmith Working on it.

kbsezginel committed:
Merge branch 'master' of https://github.com/MolSSI-SSS/SSS_2017_QuESt

kbsezginel committed:
Merge branch 'master' of https://github.com/SrinivasMushnoori/SSS_2017_QuESt

kbsezginel committed:
Deleted unnecessary lines from parameters

Commit:
Merge pull request #3 from kbsezginel/master

Deleted unnecessary lines from parameters
